File: Python Excel Examples/RangesApiDemo/unMergeRange.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.ranges_api.RangesApi(configuration)

name = "UnMergeRange.xlsx"
folder = "Ranges"
storage = ""
sheetName = "Sheet1"
Range = spirecloudexcel.models.Range()
Range.first_row = 3
Range.first_column = 4
Range.row_count = 2
Range.column_count = 5
api.unmerge_range(name, sheetName, Range, storage=storage, folder=folder)

File: JavaScript Excel Examples/RangesApiDemo/unMergeRange.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var SpirecloudExcel = require('../../src/index');
    var apiClient = new SpirecloudExcel.ApiClient(appId, appKey,baseUrl);
    var instance = new SpirecloudExcel.RangesApi(apiClient);
    
    var name = "unmergeRange.xlsx";
    var opts = { folder: "input" };
    var sheetName = "Sheet1";
    var range = new SpirecloudExcel.Range();
    //The start value is "1"
    range.FirstRow = 3;
    //The start value is "1"
    range.FirstColumn = 3;
    range.ColumnCount = 8;
    range.RowCount = 5;
    instance.unmergeRange(name, sheetName, range, opts,
        function (error, data, response) {
        if (error) throw error;
    });
})();